packages/adapter-cloudflare/index.js

```diff
@@ -57,7 +57,8 @@ export default function (options = {}) {
 				bundle: true,
 				loader: {
 					'.wasm': 'copy'
-				}
+				},
+				external: ['cloudflare:email']
```

Comment:
I wonder if it'd be more correct and future-proof to use `*` here: https://esbuild.github.io/api/#external
```suggestion
				external: ['cloudflare:*']
```

Reply:
Might as well - though I think npm packages can contain a `:` so it's technically possible we'll break something

Reply:
Virtual modules by ViteJS plugin usually have `:` in the module name.

https://vitejs.dev/guide/api-plugin.html#virtual-modules-convention

This may break in the future if Cloudflare provides a ViteJS plugin for a virtual module. 🐸

Reply:
virtual modules don't have just `:` in the name, but rather `virtual:` and/or `\0`: https://vitejs.dev/guide/api-plugin.html#virtual-modules-convention